### PR TITLE
Code Cleanup and Hi-Fi Changes.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
     "husky": "4.2.3",
+    "jest-styled-components": "7.0.2",
```
Comment (devinleighsmith) [inline frontend/package.json:85]: Fixes some issues with Jest styled component generated classnames and snapshots.

File: frontend/src/buttons.scss [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 .btn.btn-primary {
   background-color: $primary-color;
-  font-family: 'BCSans' Fallback, sans-serif;
+  font-family: 'BCSans', Fallback, sans-serif;
```
Comment (devinleighsmith) [inline frontend/src/buttons.scss:24]: Was getting a warning in Chrome about the syntax, this resolves.

File: frontend/src/components/common/form/FastCurrencyInput.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ const defaultMaskOptions = {
   thousandsSeparatorSymbol: ',',
   allowDecimal: true,
   decimalSymbol: '.',
-  decimalLimit: 2, // how many digits allowed after the decimal
+  decimalLimit: 0, // how many digits allowed after the decimal
```
Comment (devinleighsmith) [inline frontend/src/components/common/form/FastCurrencyInput.tsx:16]: This is based on a request from Mikaela to remove the cent values.

File: frontend/src/features/projects/dispose/strings.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const clickableTooltip = 'Click to view property details';
```
Comment (devinleighsmith) [inline frontend/src/features/projects/dispose/strings.ts:1]: Using this file for storage of long strings with no variable content.

File: frontend/src/fonts.scss [hunk 5]
```diff
@@ -4,17 +4,17 @@
   src: url('../node_modules/@bcgov/bc-sans/fonts/BCSans-Regular.woff') format('woff');
 }
 @font-face {
-  font-family: 'BCSans';
+  font-family: 'BCSans-Italic';
```
Comment (devinleighsmith) [inline frontend/src/fonts.scss:7]: I think this is always what I intended to do here, ensures consistent BCSans styling across the application for bold/italic.

File: backend/dal/Services/Concrete/ProjectService.cs [hunk 6]
```diff
@@ -98,40 +98,57 @@ public Project Get(string projectNumber)
...
-                .Include(p => p.Properties).ThenInclude(p => p.Building).ThenInclude(b => b.BuildingConstructionType)
-                .Include(p => p.Properties).ThenInclude(p => p.Building).ThenInclude(b => b.BuildingPredominateUse)
-                .Include(p => p.Properties).ThenInclude(p => p.Building).ThenInclude(b => b.BuildingOccupantType)
-                .Include(p => p.Properties).ThenInclude(p => p.Building).ThenInclude(b => b.Agency).ThenInclude(a => a.Parent)
-                .AsNoTracking()
                 .FirstOrDefault(p => p.ProjectNumber == projectNumber &&
                     (isAdmin || userAgencies.Contains(p.AgencyId))) ?? throw new KeyNotFoundException();
 
+            //The following reduces the load on the database compared to eager loading all parcel/building props.
+            this.Context.Entry(project)
```
Comment (Fosol) [inline backend/dal/Services/Concrete/ProjectService.cs:105]: I suspect this is a far better performing group of queries!

File: frontend/src/features/projects/dispose/strings.ts (hunk 4 above)
Comment (Fosol) [reply frontend/src/features/projects/dispose/strings.ts:1]: Makes me think we may need to start thinking about using a resource file for all string values one day.

File: frontend/src/features/projects/dispose/ProjectDisposalSubmitted.tsx [hunk 7]
```diff
@@ -3,20 +3,33 @@ import './ProjectDisposalSubmitted.scss';
...
 /**
  * Display success message after all project disposal steps are completed successfully.
  */
 export function ProjectDisposalSubmitted() {
+  const keycloak = useKeycloakWrapper();
   return (
     <Container className="ProjectDisposalSubmitted">
-      <FaRegCheckCircle size={64} />
-      <h5>Application Successfully Submitted</h5>
+      <ColoredWrapper style={{ color: '#2e8540' }}>
```
Comment (asanchezr) [inline frontend/src/features/projects/dispose/ProjectDisposalSubmitted.tsx:20]: Glad to see usage of `styled-components` here - but would like to know why the double styling? Isn't `<ColoredWrapper>` defined earlier as a div with that same color?

Comment (devinleighsmith) [reply frontend/src/features/projects/dispose/ProjectDisposalSubmitted.tsx:20]: oops

Comment (devinleighsmith) [reply frontend/src/features/projects/dispose/ProjectDisposalSubmitted.tsx:20]: I'll address this in my next PR.

**Pull request merged**